video/v1.2/video.jsonld

```diff
@@ -0,0 +1,527 @@
+      },
+      "verb": "https://w3id.org/xapi/video/verbs/seeked",
+      "objectActivityType": "https://w3id.org/xapi/video/activity-type/video",
+      "rules": [
+        {
+          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/time-to']",
+          "presence": "included"
+        },
+        {
+          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/time-from']",
+          "presence": "included"
+        }
+      ]
+    },
+    {
+      "id": "https://w3id.org/xapi/video/templates#completed",
+      "inScheme": "https://w3id.org/xapi/video/v1.2",
+      "prefLabel": {
+        "en": "Completed"
+      },
+      "definition": {
+        "en": "The statement template and rules associated with a video being completed."
+      },
+      "verb": "http://adlnet.gov/expapi/verbs/completed",
+      "objectActivityType": "https://w3id.org/xapi/video/activity-type/video",
+      "rules": [
+        {
+          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/time']",
+          "presence": "included"
+        },
+        {
+          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/progress']",
+          "presence": "included"
+        }
+      ]
+    },
+    {
+      "id": "https://w3id.org/xapi/video/templates#volumechange",
+      "inScheme": "https://w3id.org/xapi/video/v1.2",
+      "prefLabel": {
+        "en": "Volume Change Interaction"
+      },
+      "verb": "http://adlnet.gov/expapi/verbs/interacted",
+      "objectActivityType": "https://w3id.org/xapi/video/activity-type/video",
+      "rules": [
+        {
+          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/time']",
+          "presence": "included"
+        },
+        {
+          "location": "$.context.extensions['https://w3id.org/xapi/video/extensions/volume']",
+          "presence": "included"
+        }
+      ]
+    },
+    {
+      "id": "https://w3id.org/xapi/video/templates#screenchange",
+      "inScheme": "https://w3id.org/xapi/video/v1.2",
+      "prefLabel": {
+        "en": "Screen Change Interaction"
+      },
+      "verb": "http://adlnet.gov/expapi/verbs/interacted",
+      "objectActivityType": "https://w3id.org/xapi/video/activity-type/video",
+      "rules": [
+        {
+          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/time']",
+          "presence": "included"
+        },
+        {
+          "location": "$.context.extensions['https://w3id.org/xapi/video/extensions/full-screen']",
+          "presence": "included"
+        },
+        {
+          "location": "$.context.extensions['https://w3id.org/xapi/video/extensions/screen-size']",
+          "presence": "included"
+        },
+        {
+          "location": "$.context.extensions['https://w3id.org/xapi/video/extensions/video-playback-size']",
+          "presence": "included"
+        }
+      ]
+    }
+  ],
+  "patterns": [
```

Comment:
All of these need to be inScheme of the current profile version. That was mistakenly marked optional in the profile spec for Patterns.

Reply:
@fugu13 thanks! Will fix and update today!